frontend/src/components/AnalyticsOverview/StatsBox.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styles from './AnalyticsOverview.module.css';
+
+export type StatsBoxProps = {
+    icon: string,
+    stats: number,
+    description: string
+}
+
+const StatsBox = ({icon, stats, description}: StatsBoxProps) => (
+    <div className={styles.statsbox}>
+        <div className={styles.left}>
+            <img className={styles.icon} src={icon}/>
```

Comment:
the `icon` prop should only be restricted to a few values, and you should associate the `icon` value with one of the images, so we dont have to specify the file name when creating the overview